src/Microsoft.Graph.Core/Requests/Content/BatchRequestContent.cs

```diff
@@ -0,0 +1,198 @@
+﻿// ------------------------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
+// ------------------------------------------------------------------------------
+
+namespace Microsoft.Graph
+{
+    using Newtonsoft.Json;
+    using Newtonsoft.Json.Linq;
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Linq;
+    using System.Net;
+    using System.Net.Http;
+    using System.Net.Http.Headers;
+    using System.Text;
+    using System.Threading.Tasks;
+
+    /// <summary>
+    /// A <see cref="HttpContent"/> implementation to handle json batch requests.
+    /// </summary>
+    public class BatchRequestContent: HttpContent
+    {
+        private const int MAX_NUMBER_OF_REQUESTS = 20;
```

Comment:
wouldn't it be interesting to have that as a public constant somewhere? this way as a consumer of the sdk if I have a large set of things to do in a list I want to slice into batches, I could rely on that instead of having my own constant. And if in the future the number changes, just updating the SDK should be enough to be compliant with the service instead of having to dig into the documentation